tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -164,7 +164,8 @@ ZTEST_F(ascs_test_suite, test_sink_ase_read_state_idle)
 	zexpect_not_null(fixture->ase_snk.attr);
 
 	ret = ase->read(conn, ase, &hdr, sizeof(hdr), 0);
-	zassert_false(ret < 0, "attr->read returned unexpected (err 0x%02x)", BT_GATT_ERR(ret));
+	zassert_false(ret < 0, "attr->read returned unexpected (err 0x%02lx)",
+		      BT_GATT_ERR(ret));
```

Comment:
Thanks for fixing this :) I'm wondering if `lx` is the right format though; `ssize_t` is not a unsigned value, so `lx` should give the same warning as `x`, shouldn't it? 

One alternative would have been to do something like

```suggestion
	zassert_false(ret < 0, "attr->read returned unexpected (err 0x%02x)",
		      (uint8_t)BT_GATT_ERR(ret));
```

i.e. cast the value to an uint8_t, as ATT errors are only 8 bit (which is also evident from the `0x%02X`).